EKF/mag_control.cpp

```diff
@@ -327,3 +329,14 @@ void Ekf::run3DMagAndDeclFusions()
 	}
 }
 
+bool Ekf::otherHeadingSourcesHaveFinished()
+{
+    bool no_other_yaw_aiding_than_mag = noOtherYawAidingThanMag();
+
+    // detect rising edge
+    bool result = no_other_yaw_aiding_than_mag && !_no_other_yaw_aiding_than_mag;
```

Comment:
I am not so happy with the naming here. 
1. The local and global variable are easy to confound. Maybe just call `noOtherYawAidingThanMag()` in this line directly.
2. `no_other_yaw_aiding_than_mag` is a bit a misleading name, as it suggests that mag is already running. Could we change it to something like `no_non_mag_heading_aiding_running`? The same would apply for the function name.